FINT.Model.Administrasjon/FINT.Model.Administrasjon.csproj

```diff
@@ -4,8 +4,10 @@
     <TargetFramework>netstandard1.4</TargetFramework>
     <PackageRequireLicenseAcceptance>False</PackageRequireLicenseAcceptance>
     <Version>3.5.0</Version>
```

Comment:
Det stemmer at dette versjonsnummeret blir overstyrt ved publisering, ikke sant?

Reply:
Du bør også kjøre `git rm --cached -rf .idea` for å få filene bort fra Git repo.  `--cached` parameteren gjør at de kun blir fjernet fra Git, ikke fra disk.

Reply:
Da er de fjernet :-)